src/main/resources/data/crimson/recipes/crafting_crimson_planks.json

```diff
@@ -2,7 +2,7 @@
     "type": "minecraft:crafting_shapeless",
     "ingredients": [
         {
-            "item": "crimson:crimson_log"
+            "tag": "crimson:crimson_logs"
```

Comment:
may as well change the indentation to 2 spaces while you're here

Reply:
This file already uses 2 spaces for indentation.